libp2p/NodeTable.h

```diff
@@ -257,6 +254,8 @@ class NodeTable : UDPSocketEvents
     /// Looks up a random node at @c_bucketRefresh interval.
     void doDiscovery();
 
+    void doHandleTimeouts();
```

Comment:
This is handled by evictions in the discovery process and the discovery process which is every 7200ms.

Reply:
No more separate evictions mechanism...

Before my changes the nodes that didn't answer to the initial Ping were never deleted (from `m_allNodes`)

Reply:
By the way, why 7200 ms?